### DISCUSSION TITLE
How To Set Up Slow Motion Record?

### DISCUSSION THREAD

**Discussion by ClemSolo** [android/camera-samples#442] [Q&A]
Hello all,

I hope you are well.

After some hours to read the "developer.android.com" website, try to modify the Camera2SlowMotion sample, I sutck with the setup to get video with the SlowMotion setup.

First of all, I build the Camera2SlowMotion sample and I installed it on my Samsung S8. I have got the 2 choices: one with 120 FPS & one with 240 FPS. I clicked on the 240 FPS then I recorded a video. When I am plyaing the video, it works fine but the video is an high video quality with no Slow Motion effect. Fine, I checked the documentation and I saw that I have to set up:
the 

- "Video Encoding BitRate" with "setVideoEncodingBitRate" at 8 Mbps
- "Video FrameRate" with "setVideoFrameRate" at 30 FPS (because I have chosen the 240 FPS use case)

I build, installed and tried to record a video and no Slow Motion effect.

May be I am totally wrong but when I saw the video capture with the Pen, I am sure that I missed something.

Could you help me?

Thank you very much for your support and have a nice day.

Clem

**Comment by MartinalJones** [comment]
Samsung has limited third-party access to higher frame rate video.  Not sure why, but it seems rather clear that this has been done intentionally.
Typically, when using any third-party (or personally-coded app), the record will happen, but it will be at 30fps.

If you find any way to reliably get access to the 240fps camera functionality, please be sure to post here.  I have tried it on the Galaxy S8, S9, and S10 series.  Haven't tried any of the S20, S21, or S22 phones, but haven't read anything that would lead me to believe that Samsung has opened this functionality at all.
